client/lib/mixins/analytics/index.js

```diff
@@ -525,6 +525,43 @@ const EVENTS = {
 						email
 					}
 				);
+			},
+
+			pendingAccountLogInClick( { domain, user, severity, isMultipleDomains, section } ) {
+				analytics.ga.recordEvent(
+					'Domain Management',
+					`Clicked "Log in" link in Google Apps pending ToS notice in ${ section }`,
+					'Domain Name',
+					domain
+				);
+
+				analytics.tracks.recordEvent(
+					'calypso_domain_management_google_apps_pending_account_log_in_click',
+					{
+						domain,
+						user,
+						severity,
+						isMultipleDomains,
+						section
+					}
+				);
+			},
+
+			showPendingAccountNotice( site, domainName ) {
+				const siteSlug = site && site.slug;
+				analytics.ga.recordEvent(
+					'Domain Management',
+					'Showed pending account notice',
+					siteSlug
+				);
+
+				analytics.tracks.recordEvent(
+					'calypso_domain_management_google_apps_pending_account_notice_show',
+					{
+						site: siteSlug,
+						domain_name: domainName
+					}
```

Comment:
I think we should make the properties of the two events as similar as possible. We should be able to add as much detail as in the click event.